docs/guides/jsdom.md

```diff
@@ -17,7 +17,7 @@ As a result, a standalone script like the one below is generally a good approach
 var jsdom = require('jsdom').jsdom;
 
 global.document = jsdom('');
-global.window = document.defaultView;
+global.window = global;
 Object.keys(document.defaultView).forEach((property) => {
```

Comment:
I think a PR on the readme is fine for now - I don't think we have the code living anywhere else.